src/schema/type_base_string.py

```diff
@@ -1,6 +1,8 @@
 from jsl import Document
 from jsl.fields import StringField
+from snactor.registry.schemas import registered_schema
 
 
+@registered_schema('1.0')
```

Comment:
ahh i don't know if it's a good idea to use this BaseTypeString etc. it doesn't work without registered_schema, that's why i added this decorator